frontend/src/pages/Notifications/NotificationPageContainer.js

```diff
@@ -35,7 +43,9 @@ const mapDispatchToProps = (dispatch, props) => {
     markMultipleNotificationsAsRead: (notificationIds, offset, limit) =>
       dispatch(markMultipleNotificationsAsRead(notificationIds, offset, limit)),
     setNotifcationsPerPage: limit => dispatch(setNotifcationsPerPage(limit)),
-    setNotificationOffset: offset => dispatch(setNotificationOffset(offset))
+    setNotificationOffset: offset => dispatch(setNotificationOffset(offset)),
+    enableLiveUpdates: isEnabled => dispatch(enableLiveUpdates(isEnabled)),
```

Comment:
`enableLiveUpdates` --> `setIsLiveUpdatesEnabled`?

Reply:
I split the action into enable/disable live Updates.